### PR TITLE
Footnotes link styling fix

### LINKED ISSUE #5789: Footnotes link fix
For footnotes, I think only the linked portion should be underlined now (hover state blue) so default is just grey no underline (currently if it is linked the whole footnote is blue) Sabrina revised the [design sketch files](https://share.goabstract.com/c5b5156f-9bb9-4147-a45d-0bf020c0c76b?collectionLayerId=442fbbfa-febf-40ce-b3ff-918fcaa50fdb&mode=design) to reflect this

Example of what it should be like:

Criteria:
- [ ] Make sure the entire footnote isn't a link. 
- [ ] Ensure links inside the footnote are still underlined 
- [ ] Leave the numbering (to the left of a footnote) as a link

### PR DESCRIPTION
**Pull request by KalobTaulien**
Closes #5789 

Criteria:
- [x] Make sure the entire footnote isn't a link. 
- [x] Ensure links inside the footnote are still underlined 
- [x] Leave the numbering (to the left of a footnote) as a link

Review app: https://foundation-s-5789-footn-vzyrnp.herokuapp.com/en/
Test page: https://foundation-s-5789-footn-vzyrnp.herokuapp.com/en/publication-page-with-child-article-pages/meeting-measure-trial-exist-stop-trial-interest/

**Note:** Changed the `<a>` to a `<div`> in the template so `|richtext` could render child `<a>` elements. This was causing the entire section to be a link when it shoudn't have been. 

Visual changes went from this:
image

To this:
image

### CODE REVIEW

**Comment by mmmavis** [comment]
@sabrinang I think test pages are on

- https://foundation-s-5789-footn-vzyrnp.herokuapp.com/en/publication-page-with-child-article-pages/
- https://foundation-s-5789-footn-vzyrnp.herokuapp.com/en/publication-page-with-chapter-pages/

File: network-api/networkapi/templates/fragments/footnotes.html [hunk 1]
```diff
@@ -12,9 +12,9 @@ <h3 class="footnotes-title">{% trans "Footnotes" %}</h3>
           <a href="#footnote-source-{{ forloop.counter }}" class="footnote-number mr-3" aria-label="{% trans 'Back to content' %}">
             [{{ forloop.counter }}]
           </a>
-          <a href="#footnote-source-{{ forloop.counter }}" class="footnote-text" aria-label="{% trans 'Back to content' %}">
+          <div href="#footnote-source-{{ forloop.counter }}" class="footnote-text" aria-label="{% trans 'Back to content' %}">
```
Comment (mmmavis) [inline network-api/networkapi/templates/fragments/footnotes.html:15]:
```suggestion
          <div class="footnote-text">
```

**Comment by KalobTaulien** [comment]
@mmmavis @sabrinang Test page here:
https://foundation-s-5789-footn-vzyrnp.herokuapp.com/en/publication-page-with-child-article-pages/meeting-measure-trial-exist-stop-trial-interest/

File: source/sass/components/article-page.scss [hunk 2]
```diff
@@ -71,12 +71,20 @@
...
   @include set-text-size(18px, 27px);
   font-family: $font-family-sans-serif;
   font-weight: normal;
-  text-decoration: underline;
+
+  a {
+    text-decoration: underline;
+    color: $gray-60;
+
+    &:hover {
```
Comment (mmmavis) [inline source/sass/components/article-page.scss:84]:
```suggestion
    color: inherit;

    @include hover-focus-active {
```

**Review by mmmavis — CHANGES_REQUESTED** [review]

**Review by sabrinang — APPROVED** [review]
Thanks for making the footnote text not linked and leaving URLs underlined :+1:

File: network-api/networkapi/templates/fragments/footnotes.html (hunk 1 above)
Comment (KalobTaulien) [reply network-api/networkapi/templates/fragments/footnotes.html:15]: Haha you mean <div>'s dont have hrefs? 👀 Thanks for catching this!

**Comment by KalobTaulien** [comment]
@mmmavis Your changes were merged in (thank you for the code suggestions!) and I tested it locally to double check it looks like the design. Ready for re-review.

**Review by mmmavis — APPROVED** [review]
🎉

**Pull request merged**